public-app-getting-started-template/README.md

```diff
@@ -5,3 +5,17 @@ This is a project containing a public app, so developers can get up and running
 ## Requirements
 
 TBD
+
+## Set up
+Once you run `hs project upload` and the projects gets fully built and deployed onto your developer portal. You want to then install the app to a portal of your choice via the install URL located in your App's auth tab.
```

Comment:
Suggested text edits:

# Set up

1. Run `hs project upload --account=<dev-acct-id>` to build and deploy the project to your developer account. 
2. Retrieve the install URL located in your app's auth tab. 
3. Use that install URL to install the app in a developer test account of your choice.